.github/workflows/posix.yml

```diff
@@ -33,9 +33,11 @@ jobs:
           [actions-38.yaml, "not slow and not clipboard", "", "", "", "", ""],
           [actions-38.yaml, "slow", "", "", "", "", ""],
           [actions-pypy-38.yaml, "not slow and not clipboard", "", "", "", "", "--max-worker-restart 0"],
-          [actions-39-numpydev.yaml, "not slow and not network", "xsel", "", "", "deprecate", "-W error"],
           [actions-39.yaml, "slow", "", "", "", "", ""],
-          [actions-39.yaml, "not slow and not clipboard", "", "", "", "", ""]
+          [actions-39.yaml, "not slow and not clipboard", "", "", "", "", ""],
+          [actions-310-numpydev.yaml, "not slow and not network", "xclip", "", "", "deprecate", "-W error"],
```

Comment:
@lithomas1 Why was xclip chosen here instead of xsel?

I think there's this flaky clipboard test now on this build, and the only difference between this and the other clipboard ones is xclip vs xsel

https://github.com/pandas-dev/pandas/runs/4875566178?check_suite_focus=true#step:10:51